great_expectations/data_context/store/store.py

```diff
@@ -154,3 +153,7 @@ def self_check(self, pretty_print):
         NotImplementedError(
             f"The test method is not implemented for Store class {self.__class__.__name__}."
         )
+
+    @property
+    def config(self) -> dict:
+        raise NotImplementedError
```

Comment:
Can't this just return the config, and we save the config in __init__ instead of using the filter active running stuff?